App/View Controllers/Posts/PostsPageViewController.swift

```diff
@@ -637,7 +637,7 @@ final class PostsPageViewController: ViewController {
             }
         }()
         
-        composeItem.isEnabled = !thread.closed
+        composeItem.isEnabled = !thread.closed && thread.forum?.canPost == true
```

Comment:
I'm not sure about this; we should treat "replying to a thread" as a different permission from "making a new thread". A forum's "can post" means "make new thread".

(We should also figure out a better way to disable replying than "thread is closed", because some users can post to closed threads. But that's not in scope for this PR.)